package.json

```diff
@@ -67,32 +68,28 @@
   },
   "scripts": {
     "dev": "nodemon src/index.ts",
-
     "test": "jest --coverage --runInBand",
     "coverage": "cat coverage/lcov.info | coveralls",
     "coveralls": "jest --coverage && cat ./tests/coverage/lcov.info | coveralls",
     "lint:src": "tslint -c tslint.json 'src/**/*.ts'",
     "lint:test": "tslint -c tslint.json 'test/**/*.ts'",
     "lint": "yarn lint:src && yarn lint:test",
-
     "stash": "git stash -u --keep-index",
     "unstash": "git stash pop",
     "check-unstaged": "bash scripts/check_unstaged.sh",
-
+    "postinstall": "yarn upgrade-validations",
     "build": "tsc --project .",
     "build-migrations": "yarn clean:compiled-migrations && tsc -p src/migrations",
     "clean": "rm -rf dist",
     "clean:compiled-migrations": "rm -rf dist_migrations",
-
+    "upgrade-validations": "yarn upgrade validations && cd node_modules/validations && yarn build",
```

Comment:
mmmmmmmmm... quedo medio sucio esto, tener que entrar a node modules y hacer build

https://cameronnokes.com/blog/the-30-second-guide-to-publishing-a-typescript-package-to-npm/

tal vez lo mas sencillo sea meter a npm en medio, como en el repo de tsconfig

Reply:
Te referis a publicarlo como libreria? Porque para que usar npm? no entendi